content/departments/engineering/index.md

```diff
@@ -51,6 +51,34 @@ The following functional teams work across the above dev teams:
 - [Working with issues](working-with-issues.md)
 - [Our software development lifecycle (SDLC)](sdlc.md)
 
+## What's in a feature?
+
+For every feature we ship, consider:
```

Comment:
Is this an ordered list?

Reply:
No (except that putting security first was intentional).